pkg/authorization/apiserver/apiserver.go

```diff
@@ -89,8 +92,15 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 		return nil, err
 	}
 
+	v1alpha1Storage, err := c.v1alpha1RESTStorage()
+	if err != nil {
+		return nil, err
+	}
+
 	apiGroupInfo := genericapiserver.NewDefaultAPIGroupInfo(authorizationapiv1.GroupName, c.ExtraConfig.Scheme, metav1.ParameterCodec, c.ExtraConfig.Codecs)
-	apiGroupInfo.VersionedResourcesStorageMap[authorizationapiv1.SchemeGroupVersion.Version] = v1Storage
+	apiGroupInfo.PrioritizedVersions = []schema.GroupVersion{authorizationapiv1.GroupVersion, authorizationapiv1alpha1.GroupVersion}
```

Comment:
See what this does when normally run.  The alpha resource should be off and I wouldn't expect it to appear in discovery